### ISSUE TITLE
allow image upload when creating card

### ISSUE THREAD

**Issue by marydavis** [waffleio/waffle.io#3169]

**Comment by homeyer** [comment]
what are you thoughts on the future of new issue creation once we have a different type of detail view? the AZ way was to reuse the same format, right? 

That's down the road of course, just wondering if we should wait on this until then.

**Comment by marydavis** [comment]
the reuse of the sidebar was too confusing, ended up going back to similar to what we have now in Waffle.

**Comment by homeyer** [comment]
oh cool, good to know!

**Comment by AnaMSteele** [comment]
Closing this issue since we haven't seen any community requests or upvotes and are prioritizing other more requested items in the near term. Please feel free to continue to comment if this is relevant to you and your team. Thank you.
